tests/test_cli.py

```diff
@@ -15,14 +15,12 @@ def root_dir() -> Path:
     return Path(__file__).parent.parent
 
 
+@pytest.mark.skipif(not _check_edf2asc_present(), reason="edf2asc missing")
 @pytest.mark.parametrize("metadata_file", [data_dir() / "metadata.yml", None])
 @pytest.mark.parametrize("output_dir", [data_dir() / "output", None])
 @pytest.mark.parametrize("use_relative_path", [False, True])
 def test_edf_cli(use_relative_path, metadata_file, output_dir, eyelink_test_data_dir):
-    if not _check_edf2asc_present():
-        pytest.skip("edf2asc missing")
-
-    input_dir = eyelink_test_data_dir / "decisions"
+    input_dir = eyelink_test_data_dir / "satf"
```

Comment:
There is one EDF file in the "decision" dataset that fails.
Need to figure out why: maybe not in this PR though